test/e2e/common_test.go

```diff
@@ -56,8 +56,8 @@ func defaultShoot(generateName string, blackholing bool, blockAddress string) *g
 			SecretBindingName: ptr.To("local"),
 			CloudProfileName:  ptr.To("local"),
 			Kubernetes: gardencorev1beta1.Kubernetes{
-				Version:                     "1.26.0",
-				EnableStaticTokenKubeconfig: ptr.To(true),
+				Version:                     "1.27.1",
+				EnableStaticTokenKubeconfig: ptr.To(false),
```

Comment:
The `enableStaticTokenKubeconfig `field is anyways deprecated - we could remove it instead of setting it to false.